egressassess_sg.tf

```diff
@@ -0,0 +1,55 @@
+# Security group for the Egress-Assess instances
+resource "aws_security_group" "egressassess" {
+  provider = aws.provisionassessment
+
+  vpc_id = aws_vpc.assessment.id
+
+  tags = {
+    "Name" = "Egress-Assess"
+  }
+}
+
+# Allow egress to anywhere via HTTP and HTTPS
+#
+# For: Package downloads and updates
```

Comment:
Is this egress also for the [Auto-Egress-Assess](https://github.com/tarrell13/Auto-Egress-Assess) tool? If so we should explicitly mention it here. Otherwise should there be other egress rules for that tool's functionality?

Reply:
There are no additional egress rules needed on the Egress-Assess instance.

Reply:
This egress is for the Auto-Egress-Assess tool.  Auto-Egress-Assess is the new name of their version of the Egress-Assess tool.  They "worked around" the issue of merging back into the upstream by changing their fork into a new repo.

Reply:
@m1j09830 I'm not really understanding your response. If this rule were to be removed would the [Auto-Egress-Assess](https://github.com/tarrell13/Auto-Egress-Assess) tool still function?

Reply:
Yes, the tool would still function- it does not require any egress ports.  80 and 443 are only used for pulling in packages. 

The ports that the tool listens on are opened via `var.inbound_ports_allowed`:
https://github.com/cisagov/cool-assessment-terraform/blob/2dd006e6228253e22e802393c749876bd080c1b1/variables.tf#L136-L164

Reply:
@mcdonnnj does @dav3r's response make more sense? I apologize if I'm not being clear but he covered what I was trying to say much more eloquently than I could.

Reply:
Went and RTFM of the tool so I have a better understanding of what it does. Assessors are (I'm assuming) only using the server portion of the code on this instance and thus only need to set ingress rules for the tool to function.